elastic/kibana
# Forward Port 7.11 autocomplete definitions to 7.x

# Summary
This PR is to update the autocomplete suggestions for new API endpoints and parameters introduced in 7.11 into the 7.x Branch.

# How to test
Navigate to the Dev Console.
Enter the different endpoints that were updated in the PR.
After adding the endpoint for the request, check to see if the different parameters (whether URL or Body) show up in their respective places.
